lib/main.js

```diff
@@ -161,42 +161,130 @@ ava.test.serial(`resin host OS update [${options.resinOSVersion} -> ${options.re
 })
 
 if (!process.env.CI) {
-  ava.test('experience: get a shell into a running application container using the Resin CLI', async (test) => {
+  ava.test('connectivity: Provision on Ethernet', async (test) => {
+    console.log('\n\nConnectivity: Provision on Ethernet')
```

Comment:
We shouldn't be adding these `console.log`s. If anything should be part of the manual test case info, it should be implemented inside `utils.runManualTestCase`